drivers/usbdev/rndis.c

```diff
@@ -2334,9 +2356,17 @@ static int usbclass_setup(FAR struct usbdevclass_driver_s *driver,
                   break;
 #endif
 
+#ifdef CONFIG_USBDEV_DUALSPEED
+                case USB_DESC_TYPE_OTHERSPEEDCONFIG:
+#endif /* CONFIG_USBDEV_DUALSPEED */
                 case USB_DESC_TYPE_CONFIG:
                   {
+#ifndef CONFIG_USBDEV_DUALSPEED
```

Comment:
```suggestion
#ifdef CONFIG_USBDEV_DUALSPEED
```

Reply:
The speed and type arguments are only available if CONFIG_USBDEV_DUALSPEED is defined.

Reply:
My mean is changing the order not remove #ifdef/#endif.